cypress/integration/campaign-info-block.js

```diff
@@ -140,6 +141,20 @@ describe('Campaign Info Block', () => {
     });
   });
 
+  /** @test */
+  it('Visit campaign, as an authenticated user, show scholarship modal without "Apply Now" ', () => {
```

Comment:
How's about:
```suggestion
  it('Visit campaign, as an affiliated user, show scholarship modal without "Apply Now" ', () => {
```
Since we're trying to explicitly test the signup button when they're affiliated?